.github/workflows/podcast-loadtest.yml

```diff
@@ -0,0 +1,57 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Podcast Load Test
+
+# Controls when the workflow will run
+on:
+  workflow_dispatch:
+
+env:
```

Comment:
I am a bit worries about these hardcoded strings, we should pull from github secrets and document it.